package.json

```diff
@@ -69,7 +71,7 @@
     "loader.js": "^4.2.3",
     "mocha": "^5.0.0",
     "qunit-dom": "^0.7.1",
-    "typescript": "^2.9.2"
+    "typescript": "^3.2.2"
```

Comment:
Is it necessary to drop support for TS 2.9?

Reply:
I don't think it is required. Do you think I should revert this or add an ember-try scenario using 2.9 instead?

_Maybe_ the upgrade caused https://github.com/rwjblue/sparkles-component/pull/22#discussion_r240713626.

Reply:
we just shouldn't make this change arbitrarily. Although it's a devDependency, the output is read by consumers. Bumping up to 3.2 would allow the accidental use of syntax that would break .d.ts files for 2.9 users.

Reply:
Downgraded to `^2.9.2`.